hack/verify-typecheck.sh

```diff
@@ -30,8 +30,13 @@ cd "${KUBE_ROOT}"
 
 make --no-print-directory -C "${KUBE_ROOT}" generated_files
 
+# As of June, 2020 the typecheck tool is written in terms of go/types, but that
+# library doesn't work well with modules.  Guidance is to rewrite tools against
+# golang.org/x/tools/go/packages.  Until that is done, force this tooling to
+# run in a fake GOPATH.
```

Comment:
Created https://github.com/kubernetes/kubernetes/issues/92253 to track this.

Will let @liggitt lgtm when he's back.

Reply:
Pushing a fix, but with caveats.  Will update the PR comment here.  PTAL as if from scratch